principal_travel/templates/travel/trip_list.html

```diff
@@ -26,6 +26,8 @@ <h4 class="panel-title">
   </div>
 </div>
 
+<h2> {{ quick_dates_2 }}</h2>
+
```

Comment:
I have no idea why that is there. It isn't on my current file and I just tried to push to origin and it said it was up to date...